pages/events/index.vue

```diff
@@ -336,6 +359,12 @@ export default class EventsPage extends QiskitPage {
     }
   }
 
+  &__calendar{
+    margin-bottom: $spacing-06;
```

Comment:
for consistency with other tabs, I think this bottom spacing should equal 64px / 4rem. Maybe we can go with `$spacing-10`